docs/user/troubleshooting-guides/03-40-api-rule-troubleshooting.md

```diff
@@ -159,9 +183,13 @@ spec:
 
 ---
 ## Configuration of `noop`, `allow`, and `no_auth` Handlers 
+
+> [!WARNING]
+> The `noop` and `allow` handlers are supported only in version `v1beta1` of APIRule, which will become deprecated in 2024. Migrate to version `v1beta2`.
+
 ### Cause
 
-In the following APIRule, the `noop` handler has the **trusted-issuers** field configured:
+In the following APIRule, the `no_auth` handler has the **trusted-issuers** field configured:
 
 ```yaml
 spec:
```

Comment:
This spec refers to v1beta1, so it might be confusing for v1beta2. Should we have all of those specific examples of APIRules in tabs for v1beta1 and v1beta2?

Reply:
Right, I didn't realize. It would be a lot of tabs for one doc so I'd say it's better to have two separate guides for v1beta1 and v1beta2. Are all the issues that this guide contains also relevant for v1beta2?

Reply:
Ok, I think everything except `Configuration of noop, allow, and no_auth Handlers` should be relevant.